binaries/Readme.md

```diff
@@ -128,33 +130,56 @@ which is why we need different binaries for those as well.
 ### How?
 
 We build multiple binaries on common operating systems with different combinations
-of OpenSSL versions. This results in a few binaries which work on a large selection
-of operating systems, distributions and cloud platforms.
+of OpenSSL versions. This results in a few binaries, which work on a large selection
+of operating systems, distributions, and cloud platforms.
 
 Initially, we built specific binaries for given platforms (e.g. Netlify), which resulted in many 
 binaries which couldn't be shared with similar platforms and it was hard to maintain. 
 
-### Binary builds
+### Binaries
 
 The following table lists the pre-built binaries provided by Prisma.
-These include Windows and Mac, and multiple variations for Linux distributions
+These include Windows and Mac, and multiple variations for Linux distributions.
 
-| Build          | OS                  | OpenSSL variations | Downloads |
+#### S3 URL schema
```

Comment:
This section already covers the `URL Scheme` section further below (which also contains outdated info). Please remove the `URL Scheme` section.